sites/bizbash.com/config/navigation.js

```diff
@@ -6,6 +6,9 @@ const topics = [
   { href: '/bizbash-lists', label: 'BizBash Lists' },
 ];
 
+let hamburgerMenu = topics;
+hamburgerMenu = [...hamburgerMenu, { href: '/industry-buzz', label: 'Industry Buzz' }];
+
```

Comment:
```suggestion
const hamburgerMenu = [...topics, { href: '/industry-buzz', label: 'Industry Buzz' }];

```

There's no need to create a new variable and then modify it -- just define the variable using const and spread the existing items into it instead. Review the [prefer-const](https://eslint.org/docs/rules/prefer-const) ESLint rule for more info.

Ideally, this should be done on L128/L131, as there's no need to define this variable at all. It's only used once

Reply:
```js
// ...
    {
      label: 'Find Inspiration',
      items: [
        ...topics,
        { href: '/industry-buzz', label: 'Industry Buzz' }
      ],
    },
    {
      label: 'Other Resources',
// ...
```